go.mod

```diff
@@ -72,12 +72,12 @@ require (
 	golang.org/x/sys v0.21.0
 	google.golang.org/grpc v1.64.0
 	google.golang.org/protobuf v1.34.2
-	k8s.io/api v0.30.2
-	k8s.io/apimachinery v0.30.2
-	k8s.io/client-go v0.30.2
-	k8s.io/cri-api v0.31.0-alpha.0.0.20240529224029-3a66d9d86654
-	k8s.io/klog/v2 v2.130.0
-	k8s.io/kubelet v0.30.2
+	k8s.io/api v0.31.0-alpha.2
+	k8s.io/apimachinery v0.31.0-alpha.2
+	k8s.io/client-go v0.31.0-alpha.2
+	k8s.io/cri-api v0.31.0-alpha.2
```

Comment:
@saschagrunert Does this version have all the changes specific to https://github.com/kubernetes/kubernetes/pull/117842?
https://github.com/cri-o/cri-o/commit/c931b90a3b4cc3b8b57043c61cca1a05fc977341#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R78

Reply:
@sohankunkerkar yes.